ccd-definition/CaseField.json

```diff
@@ -63,6 +63,14 @@
     "FieldType": "StatementOfTruth",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "claimIssuedDate",
+    "Label": "Claim issued date",
+    "FieldType": "Date",
+    "SecurityClassification": "Public"
+  },
```

Comment:
@swalker125 - do we end up having two dates for the claim issue?  One for the actual issue and one for the 'legal one' i.e. the 2 days after if it were posted? If so, I think we need to be clear about what we name them.

Reply:
claim issued is just the day it was made in ccd, confirmation of service is separate